external/sql/storm-sql-core/src/codegen/includes/parserImpls.ftl

```diff
@@ -83,4 +83,23 @@ SqlNode SqlCreateTable() :
         input_format_class_name, output_format_class_name, location,
         tbl_properties, select);
     }
+}
+
+/**
+ * CREATE FUNCTION functionname AS 'classname'
+ */
+SqlNode SqlCreateFunction() :
+{
+    SqlParserPos pos;
+    SqlIdentifier functionName;
+    SqlNode className;
+}
+{
+    <CREATE> { pos = getPos(); }
+    <FUNCTION>
+    functionName = CompoundIdentifier()
+    <AS>
+    className = StringLiteral() {
```

Comment:
That way the clauses might result into a parsing error. Instead of reporting a parsing error, it might be more user friendly to throw an UnimplementedException when compiling the SQL statements. It is your call.